modules/standard/Regex.chpl

```diff
@@ -604,6 +578,95 @@ record regex {
     this.exprType = exprType;
   }
 
+  /*
+     Initialize a compiled regular expression. This routine will throw a
+     class:`BadRegexError` if compilation failed.
```

Comment:
I'd like to tweak the wording along the lines of:
Initialize --> Initializer for
This routine will throw--> `new regex(...)` throws
failed --> fails

Please add a leading `:` to `class:'BadRegexError'`